lib/Backend/GlobOpt.cpp

```diff
@@ -6655,6 +6655,7 @@ GlobOpt::OptConstFoldBranch(IR::Instr *instr, Value *src1Val, Value*src2Val, Val
         }
         else
         {
+#pragma prefast(suppress: __WARNING_LOGICALORNONZERO, "silence unnecessary error in release build, where CONFIG_FLAG(OOPJITMissingOpts) is #defined to be 0")
```

Comment:
>__WARNING_LOGICALORNONZERO [](start = 26, length = 26)

Don't like this suppress thing. Is there another way to modify the check? May be see can we use CONFIG_FLAG_RELEASE

Reply:
I'm not a huge fan of the suppressions either, but unfortunately, `CONFIG_FLAG_RELEASE` turns out not to compile (at least in an x86fre razzle build):
`e:\git\os\src\onecoreuap\inetcore\jscript\core\lib\backend\lower.cpp(3549) : error C2039: 'OOPJITMissingOpts': is not a member of 'Js::ConfigFlagsTable'`